src/reldep.h

```diff
@@ -39,7 +39,7 @@ void hy_reldeplist_free(HyReldepList reldeplist);
 void hy_reldeplist_add(HyReldepList reldeplist, HyReldep reldep);
 int hy_reldeplist_count(HyReldepList reldeplist);
 HyReldep hy_reldeplist_get_clone(HyReldepList reldeplist, int index);
-
+HyReldepList reldeplist_from_str(HySack sack, const char *reldep_str);
```

Comment:
Yes, be careful about every `.h` file change. There is a big chance that any such change will upset at least Hughsie ;)